docs/sample_config.yaml

```diff
@@ -1352,3 +1352,18 @@ password_config:
 #  module: "my_custom_project.SuperRulesSet"
 #  config:
 #    example_option: 'things'
+
+
+## Tracer ##
+
+#tracer:
+#  # Enable / disable tracer
+#  tracer_enabled: false
+#  # The list of users who's requests will be traced
+#  # The list is a list of regex which is matched against the user_id
+#  user_whitelist:
+#    - "*"
+#  # The list of homeservers we wish to trace across
+#  # The list is a list of regex which is matched against the homeserver name
+#  homeserver_whitelist:
+#    - "*"
```

Comment:
Let's not add unused options for now, could you either remove it from this PR or add support for it?

Reply:
I've implemented whitelisting servers to which we send spans contexts to and removed whitelisting of users for the moment. Choosing a good location for user whitelisting and finding a way to abort spans needs some consideration first.